src/Discovery/aggMDSUtils.jsx

```diff
@@ -0,0 +1,124 @@
+import { discoveryConfig, hostname } from '../localconf';
+
+const aggMDSURL = `${hostname}mds/aggregate`;
+const aggMDSDataURL = `${aggMDSURL}/metadata`;
```

Comment:
minor: I'd prefer setting these 2 URLs in `localconf` and import them into here, just like we do for all other pages